SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialReceiver.h

```diff
@@ -144,8 +144,9 @@ class USpatialReceiver : public UObject
 	void RemoveActor(Worker_EntityId EntityId);
 	void DestroyActor(AActor* Actor, Worker_EntityId EntityId);
 
-	AActor* GetOrCreateActor(improbable::UnrealMetadata* UnrealMetadata, UClass* ActorClass, improbable::SpawnData* SpawnData);
-	AActor* CreateActor(improbable::SpawnData* SpawnData, UClass* ActorClass);
+	AActor* TryGetOrCreateActor(improbable::UnrealMetadata* UnrealMetadata, improbable::SpawnData* SpawnData);
+	AActor* CreateActor(improbable::UnrealMetadata* UnrealMetadata, improbable::SpawnData* SpawnData);
+	AActor* SpawnActor(improbable::SpawnData* SpawnData, UClass* Class);
```

Comment:
nit: I don't think the SpawnActor function is necessary. It's functionality bleeds into CreateActor and creates a confusing interface.

Reply:
hmmmmmmMMMMMMMMMMMMM fair point.